manifests/service.pp

```diff
@@ -28,53 +28,49 @@
   $catalina_home  = undef,
   $catalina_base  = undef,
   $use_jsvc       = false,
+  $use_init       = false,
   $java_home      = undef,
   $service_ensure = running,
   $service_enable = undef,
-  $use_init       = false,
   $service_name   = undef,
   $start_command  = undef,
   $stop_command   = undef,
-  $user           = $::tomcat::user,
+  $user           = undef,
 ) {
+  include tomcat
+  $_user = pick($user, $::tomcat::user)
```

Comment:
Is this pick necessary? It looks just just another way to default `$user` to `$::tomcat::user`.